packages/ckeditor5-engine/src/view/treewalker.ts

```diff
@@ -54,7 +56,7 @@ export default class TreeWalker {
 			 *
 			 * @error view-tree-walker-unknown-direction
 			 */
-			throw new CKEditorError( 'view-tree-walker-unknown-direction', options.startPosition, { direction: options.direction } );
+			throw new CKEditorError( 'view-tree-walker-unknown-direction', options, { direction: options.direction } );
```

Comment:
The `context` param is used to check which instance of the editor caused that exception. I'm not sure if the original implementation was valid but the new one might be broken? Is it possible to track the connection between that object and editor instance?